components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -142,6 +143,9 @@ - (void)viewDidLoad {
 - (void)viewWillLayoutSubviews {
   [super viewWillLayoutSubviews];
 
+  if (self.tableView.contentSize.height > (CGRectGetHeight(self.view.bounds) / 2)) {
+    self.mdc_bottomSheetPresentationController.preferredSheetHeight = [self openingSheetHeight];
+  }
```

Comment:
Do we need to reset this to 0 in an else case?

Reply:
No because then it will open to 0 height, we only set it if their is too many cells to fit on half the screen or more, the else case is covered in MDCPresentationController.

Reply:
My guess then is that if there aren't more cells than will fit on the screen, then we ignore preferredSheetHeight - is that correct?

If that's the case I think it might still be wise for us to be explicit and reset preferredSheetHeight to 0 when we don't expect to use it. Otherwise, we're depending on code in another file implementing behavior that might change in an unexpected way someday.

Reply:
Done